security.rst

```diff
@@ -1846,7 +1846,53 @@ To enable logging out, activate the  ``logout`` config parameter under your fire
             ;
         };
 
-Next, you need to create a route for this URL (but not a controller):
+``path`` can be either a route name, or a URI.
+
+Since Symfony 6.4, you can leverage the
```

Comment:
We should remove `Since Symfony 6.4,` in sentence. Upmerging in 7.0 is with only the drop of the versionadded directive